charts/apisix/Chart.yaml

```diff
@@ -31,7 +31,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.2.0
+version: 1.2.1
```

Comment:
Due to the large span of etcd version upgrades, I tend to upgrade a large version, how about we upgrade the version number to 1.3.0?
```suggestion
version: 1.3.0
```

Reply:
Hello @tao12345666333 

Thank you for the feedback. I updated the Chart version to `1.3.0`.